crates/extensions/c8y_auth_proxy/Cargo.toml

```diff
@@ -29,14 +29,13 @@ tokio = { workspace = true, features = [
     "process",
     "io-util",
 ] }
+tokio-tungstenite = { version = "0.20.0", features = ["rustls-tls-native-roots"] }
 tracing = { workspace = true }
 url = { workspace = true }
 
 [dev-dependencies]
 env_logger = { workspace = true }
 mockito = { workspace = true }
 rcgen = { workspace = true }
+rustls = { workspace = true, features = ["dangerous_configuration"] }
```

Comment:
What's this `dangerous_configuration` feature? By the way, `rustls` is not used.

Reply:
`dangerous_configuration` allows me to disable certificate validation for the websocket connection (for testing, hence why it's in `dev-dependencies`). If we don't like this, it can be likely be changed to instead either trust the generated certificate or use a non-secure connection (although that is a little bit tricky without duplicating a bunch of test setup code).

Reply:
> If we don't like this

That's okay.